test/common.js

```diff
@@ -18,10 +19,38 @@ if (process.env.TEST_THREAD_ID) {
 }
 exports.tmpDir = path.join(exports.testDir, exports.tmpDirName);
 
-exports.opensslCli = path.join(path.dirname(process.execPath), 'openssl-cli');
+var opensslCli = null;
+
+
+function opensslCliGetter() {
```

Comment:
Maybe add a `if (opensslCli !== null) return opensslCli;` at the top?  Avoids multiple invocations of the openssl binary.